src/pages/lock-exception/AddLockExceptionForm.js

```diff
@@ -119,125 +172,113 @@ class AddLockExceptionForm extends Component {
+                    data-test="add-lock-exception-modal-org-unit-tree"
+                >
+                    {this.state.rootWithMembers ? (
+                        <OrgUnitTree
+                            root={this.state.rootWithMembers}
+                            selected={this.state.selected}
+                            currentRoot={this.state.currentRoot}
+                            initiallyExpanded={[
+                                `/${this.state.rootWithMembers.id}`,
+                            ]}
+                            memberCollection="dataSets"
+                            memberObject={this.state.dataSet.id}
+                            onSelectClick={this.handleOrgUnitClick}
+                            onChangeCurrentRoot={this.handleChangeRoot}
+                            orgUnitsPathsToInclude={this.state.orgUnitPaths}
+                        />
+                    ) : (
+                        i18n.t('Loading organisation units tree...')
+                    )}
+                </div>
+                <div className={styles.right}>
+                    <div>
+                        <div>
+                            {this.state.currentRoot
+                                ? i18n.t(
+                                      'For organisation units within {{organisationUnitName}}:',
+                                      {
+                                          organisationUnitName: this.state
+                                              .currentRoot.displayName,
+                                      }
+                                  )
+                                : i18n.t('For all organisation units:')}
+                        </div>
+                        <OrgUnitSelectByLevel
+                            levels={this.props.levels}
+                            selected={this.state.selected}
+                            currentRoot={this.state.currentRoot}
+                            onUpdateSelection={this.handleSelectionUpdate}
+                        />
+                        <OrgUnitSelectByGroup
+                            groups={this.props.groups}
+                            selected={this.state.selected}
+                            currentRoot={this.state.currentRoot}
+                            onUpdateSelection={this.handleSelectionUpdate}
+                        />
+                        <OrgUnitSelectAll
+                            selected={this.state.selected}
+                            currentRoot={this.state.currentRoot}
+                            onUpdateSelection={this.handleSelectionUpdate}
+                        />
+                    </div>
+                </div>
+            </div>
+        )
+    }
+
     render() {
         const dataSetItems = this.props.dataSets.map(dataSet => ({
             id: dataSet.id,
             name: dataSet.displayName,
-            periodType: dataSet.periodType,
+            periodType: dataSet.periodType.name,
```

Comment:
We'll want to revert this change once https://jira.dhis2.org/browse/DHIS2-11198 is resolved.